website/content/partials/telemetry-metrics/vault/core/request-limiter/ignored.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.core.limits.concurrency.ignored ((#vault-core-limits-concurrency-ignored))
+
+Metric type | Value   | Description
+----------- | ------- | -----------
+counter     | number  | Number of ignorable request errors observed by the request limiter
```

Comment:
I'm not even quite sure what an "ignorable request" is here - should we expand on this a little to be more clear?

Reply:
I think https://github.com/hashicorp/vault/pull/25557/commits/7a38c1ff7ee0448f14b115478f967947a06e704e should clarify the ambiguity here. Let me know if you think it needs a more detailed explanation.